pkg/k8s/client/cell.go

```diff
@@ -109,29 +112,69 @@ type compositeClientset struct {
 	log           logrus.FieldLogger
 	closeAllConns func()
 	restConfig    *rest.Config
+	rt            *rotatingHttpRoundTripper
+
+	lock.RWMutex
```

Comment:
I was not able to put this lock in `cell.Config` due to it assuming that all fields are passed by flags. Not sure if there is a better place for it than here.